functions.php

```diff
@@ -120,40 +120,77 @@ function panel_login_fail( $username ) {
     if ( !empty($referrer) && !strstr($referrer,'wp-login') && !strstr($referrer,'wp-admin') ) {
         $referrer = explode('?', $referrer);
         parse_str($referrer[1], $referrer_args);
-        unset($referrer_args['idilogout']); 	//unset old idilogout arg if set
-        $referrer_args['idilogin'] = "failed"; 	//let theme know there are errors
+        unset($referrer_args['idilogout']);     //unset old idilogout arg if set
+        $referrer_args['idilogin'] = "failed";     //let theme know there are errors
```

Comment:
This code has a mixed use of single quotes (') and double quotes ("). Can you check the coding convention for wordpress plugins and unify the usage? Thanks.